src/types.rs

```diff
@@ -1,15 +1,5 @@
-use ruby_sys::types;
+pub use ruby_sys::types::*;
```

Comment:
It's not _totally_ frowned upon, but some people feel more strongly than others. Especially with a `pub use`, it can be tricky.